web/moduly/finance.php

```diff
@@ -76,24 +72,38 @@
       <p>GameCon je nutné zaplatit převodem <strong>do <?=$limit?></strong>. Platíš celkem <strong><?=$castka?></strong>, variabilní symbol je tvoje ID <strong><?=$uid?></strong>.</p>
     <?php }else{ ?>
       <p>GameCon je nutné zaplatit převodem <strong>do <?=$limit?></strong>. Platíš celkem <strong><?=$castka?></strong>, přesné údaje o platbě nalezneš výše.</p>
     <?php } ?>
     <ul class="seznam-bez-okraje">
-    <li class="poznamka">Při pozdější platbě tě systém dne <strong>1.7</strong> (příp. 16.7. při pozdější přihlášce)<strong> automaticky odhlásí</strong>.</li>
-    <li class="poznamka">Pokud si aktivity (znovu) přihlásíš po 30.6., beze slevy nebo na místě pak platíš <?=$pozde?>.</li>
-    <li class="poznamka">Při plánování aktivit si na účet pošli klidně více peněz. Přebytek ti vrátíme na infopultu nebo ho můžeš využít k přihlašování uvolněných aktivit na místě.</li>
+      <li class="poznamka">Při pozdější platbě tě systém dne <strong><?php echo datum3(HROMADNE_ODHLASOVANI) ?></strong> (příp. <?php echo datum3(HROMADNE_DRUHE_ODHLASOVANI) ?> při pozdější přihlášce)<strong> automaticky odhlásí</strong>.</li>
+      <li class="poznamka">Při plánování aktivit si na účet pošli klidně více peněz. Přebytek ti vrátíme na infopultu nebo ho můžeš využít k přihlašování uvolněných aktivit na místě.</li>
+    </ul>
+  <?php } elseif(LIMIT2) { ?>
+    <?php if($u->stat()=='CZ'){ ?>
+      <p>GameCon je nutné zaplatit převodem <strong>do <?=$limit2?></strong>. Platíš celkem <strong><?=$castka?></strong>, variabilní symbol je tvoje ID <strong><?=$uid?></strong>.</p>
+    <?php }else{ ?>
+      <p>GameCon je nutné zaplatit převodem <strong>do <?=$limit2?></strong>. Platíš celkem <strong><?=$castka?></strong>, přesné údaje o platbě nalezneš výše.</p>
+    <?php } ?>
+    <ul class="seznam-bez-okraje">
+      <li class="poznamka">Při pozdější platbě tě systém dne <strong><?php echo datum3(HROMADNE_DRUHE_ODHLASOVANI) ?> automaticky odhlásí</strong>.</li>
+      <li class="poznamka">Při plánování aktivit si na účet pošli klidně více peněz. Přebytek ti vrátíme na infopultu nebo ho můžeš využít k přihlašování uvolněných aktivit na místě.</li>
     </ul>
   <?php }else{ ?>
-    <p>Období pro slevu za včasnou platbu vypršelo, zaplatit tedy můžeš převodem nebo na místě celkem <strong><?=$castka?></strong>.</p>
+     <!--TODO hláška po druhém odhlašování-->
+    <?php if($u->stat()=='CZ'){ ?>
+      <p>Zaplatit můžeš převodem nebo na místě. Platíš celkem <strong><?=$castka?></strong>, variabilní symbol je tvoje ID <strong><?=$uid?></strong>.</p>
+    <?php }else{ ?>
+      <p>Zaplatit můžeš převodem nebo na místě. Platíš celkem <strong><?=$castka?></strong>, přesné údaje o platbě nalezneš výše.</p>
+    <?php } ?>
+    <ul class="seznam-bez-okraje">
+      <li class="poznamka">Při plánování aktivit si na účet pošli klidně více peněz. Přebytek ti vrátíme na infopultu nebo ho můžeš využít k přihlašování uvolněných aktivit na místě.</li>
+    </ul>
```

Comment:
Tento soubor už se stal příliš komplikovaným (a s velkou mírou duplikace) a budeme ho tedy muset nakonec refaktorovat. Nejspíš by před tím byla potřeba nějaká lehká analýza, co a jak se tam může vypisovat. Podle toho by se navrhlo, jak to udělat. Nová verze by už asi měla použít šablonu. To si ale vezme nějaký čas, takže se to nemusí stát hned v tomto PR (nechávám na tvém uvážení, příp. vyjádření lidí na poradě, jakou dáváme merge tohoto PR prioritu).

Reply:
Jen pro úplnost, na toto byla založena karta:
https://trello.com/c/IeFD6ry4/526-2-refaktorovat-k%C3%B3d-v%C3%BDpisu-v-souboru-financephp